lua/lsp/providers/jsonls.lua

```diff
@@ -1,20 +1,195 @@
+      "commands.json",
+    },
+    url = "https://json.schemastore.org/commands.json",
+  },
+  {
+    description = "AWS CloudFormation provides a common language for you to describe and provision all the infrastructure resources in your cloud environment.",
+    fileMatch = {
+      "*.cf.json",
+      "cloudformation.json",
+    },
+    url = "https://raw.githubusercontent.com/awslabs/goformation/v5.2.9/schema/cloudformation.schema.json",
+  },
+  {
+    description = "The AWS Serverless Application Model (AWS SAM, previously known as Project Flourish) extends AWS CloudFormation to provide a simplified way of defining the Amazon API Gateway APIs, AWS Lambda functions, and Amazon DynamoDB tables needed by your serverless application.",
+    fileMatch = {
+      "serverless.template",
+      "*.sam.json",
+      "sam.json",
+    },
+    url = "https://raw.githubusercontent.com/awslabs/goformation/v5.2.9/schema/sam.schema.json",
+  },
+  {
+    description = "Json schema for properties json file for a GitHub Workflow template",
+    fileMatch = {
+      ".github/workflow-templates/**.properties.json",
+    },
+    url = "https://json.schemastore.org/github-workflow-template-properties.json",
+  },
+  {
+    description = "golangci-lint configuration file",
+    fileMatch = {
+      ".golangci.toml",
+      ".golangci.json",
+    },
+    url = "https://json.schemastore.org/golangci-lint.json",
+  },
+  {
+    description = "JSON Schema for Grafana 5.x Dashboards",
+    url = "https://json.schemastore.org/grafana-dashboard-5.x.json",
+  },
+  {
+    description = "JSON schema for the JSON Feed format",
+    fileMatch = {
+      "feed.json",
+    },
+    url = "https://json.schemastore.org/feed.json",
+    versions = {
+      ["1"] = "https://json.schemastore.org/feed-1.json",
+      ["1.1"] = "https://json.schemastore.org/feed.json",
+    },
+  },
+  {
+    description = "Packer template JSON configuration",
+    fileMatch = {
+      "packer.json",
+    },
+    url = "https://json.schemastore.org/packer.json",
+  },
+  {
+    description = "NPM configuration file",
+    fileMatch = {
+      "package.json",
+    },
+    url = "https://json.schemastore.org/package.json",
+  },
+  {
+    description = "Traefik v2 Dynamic Configuration File Provider",
+    url = "https://json.schemastore.org/traefik-v2-file-provider.json",
+  },
+  {
+    description = "JSON schema for Visual Studio component configuration files",
+    fileMatch = {
+      "*.vsconfig",
+    },
+    url = "https://json.schemastore.org/vsconfig.json",
+  },
+  {
+    description = "Resume json",
+    fileMatch = { "resume.json" },
+    url = "https://raw.githubusercontent.com/jsonresume/resume-schema/v1.0.0/schema.json",
+  },
+}
+
+local function extend(tab1, tab2)
+  for _, value in ipairs(tab2) do
+    table.insert(tab1, value)
+  end
+  return tab1
+end
+
+local extended_schemas = extend(schemas, default_schemas)
+
 local opts = {
   setup = {
     settings = {
       json = {
```

Comment:
`settings` should be moved outside of `setup` =>
```lua
local opts = {
  settings = {
    json = {
      schemas = schemas,
    },
  },
  setup = {
    commands = {
       ....
    },
  },
}
```

Reply:
Sorry. Didn't notice PR was merged.

Reply:
no worries mate, i did fix it in https://github.com/LunarVim/LunarVim/pull/1682